internal/service/service.go

```diff
@@ -0,0 +1,181 @@
+package service
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"github.com/VTerenya/employees/internal"
+	"github.com/VTerenya/employees/internal/repository"
+)
+
+type Serv interface {
```

Comment:
Idiomatic way of declaring interfaces is a package where the Interface is being used

Reply:
+